lib/travis/build/script/csharp.rb

```diff
@@ -8,9 +8,11 @@ module Build
     class Script
       class Csharp < Script
         DEFAULTS = {
-          csharp: 'mono',
+          mono: 'latest',
         }
 
+        MONO_VERSION_REGEXP = /^(\d{1}\.)(\d{1,2}\.)(\d{1,2})$/
```

Comment:
Put the `\.` outside the capturing group, so we only get the number instead of number+dot.